LeaderboardBackend/openapi.json

```diff
@@ -462,12 +462,12 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/Run"
+                  "$ref": "#/components/schemas/RunViewModel"
```

Comment:
I wonder if we could rename entities for the generated spec, so that the viewmodels won't have that "ViewModel" suffix.

You think that's even worth it?

Reply:
https://github.com/domaindrivendev/Swashbuckle.AspNetCore#customize-schema-ids
Looks easy to do but I can't say if it's worth it.

Reply:
It's fine the way it is imo.

Reply:
Yeah screw that then lol